onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -348,6 +349,7 @@ and present state are optional. Present state could appear in output even when p
       .SinceVersion(1)
       .SetDoc("Quantization of Multi-Head Self Attention.")
       .Attr("num_heads", "Number of attention heads", AttributeProto::INT)
+      .Attr("head_size", "Size of an attention head", AttributeProto::INT, static_cast<int64_t>(-1))
```

Comment:
>head_size [](start = 13, length = 9)

it is not backward compatible with this extra attribute. we need to upgrade the op.

Reply:
@yufenglee I could still load the previous models by having the default value (-1). Is there anything I need to check for the backward compatibility?

Reply:
It should be fine as it is an optional atrribute.